app_dart/lib/src/service/config.dart

```diff
@@ -47,11 +47,6 @@ class Config {
     RepositorySlug('flutter', 'plugins'),
   };
 
-  /// List of GitHub repositories that are supported by [Scheduler].
-  static Set<RepositorySlug> schedulerSupportedRepos = <RepositorySlug>{
```

Comment:
This is the key logic here. Removing it enables the scheduler to start writing other repos.